js/components/discoveryPanel.js

```diff
@@ -28,9 +26,39 @@ var DiscoveryPanel = React.createClass({
         })
       }),
       icons: React.PropTypes.objectOf(React.PropTypes.object)
+    }),
+    controller: React.PropTypes.shape({
+      sendDiscoveryClickEvent: React.PropTypes.func
     })
   },
 
+  getDefaultProps: function () {
```

Comment:
Any benefit here outside of testing? Could we mock this up in the tests instead?